mailpoet/mailpoet
# Allow multiple values in "opened" segment [MAILPOET-3950]

There is a lot of code shared with the "clicked" segment. But that is in a different [ticket](https://mailpoet.atlassian.net/browse/MAILPOET-3951). So I left some code in which will be deleted/refactored in an upcoming ticket.

[MAILPOET-3950]

[MAILPOET-3950]: https://mailpoet.atlassian.net/browse/MAILPOET-3950?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Testing instructions:
1. Go to WP Admin > MailPoet > Lists
2. Click to Add New Segment
3. Select `opened` segment type
4. Compare results from your testing with the specification from Jira ticket above (MAILPOET-3950)
5. Verify you see all the new improvements and that creating such segmentation works
